lib/controller/component/type/answers/answers.answers.js

```diff
@@ -46,10 +46,7 @@ function reduceHeadings (accumulator, answer, index, array) {
   return accumulator.concat(answer)
 }
 
-function getAnswers (componentInstance, userData, pageInstance) {
-  const parsedComponentInstance = JSON.parse(JSON.stringify(componentInstance))
```

Comment:
interesting. I wonder how manny times we do this in the codebace

Reply:
LOTS

Reply:
It's completely unnecessary here. The code beyond this only picks off two values from `componentInstance`, both booleans